### DISCUSSION TITLE
czy dobre filtry w AdGuard

### DISCUSSION THREAD

**Discussion by ghost** [MajkiIT/polish-ads-filter#21949] [General]
mam AdGuard, czy tak mam dobrze?
![1](https://github.com/MajkiIT/polish-ads-filter/assets/104568362/eaa3bdb0-2cb8-45d2-8181-a8e9cb5ec2dc)
![2](https://github.com/MajkiIT/polish-ads-filter/assets/104568362/dc0d731b-511a-447b-a2bb-1151dc49db8b)
![3](https://github.com/MajkiIT/polish-ads-filter/assets/104568362/0099e6d9-ec84-4a3a-8498-c31910a982b7)
![4](https://github.com/MajkiIT/polish-ads-filter/assets/104568362/6f9e01b1-0943-4199-827c-bf4a9db3282c)

**Comment by gzenio22** [comment]
Włącz elementy irytujące dograj z certyficate.it polskie filtry RODO ciasteczkowe oraz ja bym dograł I dont care about cookie.

Wróć poprawka wygląda to spoko moje niedopatrzenie :P

**Comment by ghost** [comment]
czyli wszystko ok, to czemu na wp.pl sa reklamy, po wlaczeniu edge?

**Comment by gzenio22** [comment]
Bo WP to stan umysłu ja aby totalnie sie pozbyć reklam z WP przeszedłem na natywną aplikacje Adguard ale ostrzegam jest ona płatna

**Comment by ghost** [comment]
aha, dobra
